### DISCUSSION TITLE
How can input and change the "parameters" value in cfclient tab??

### DISCUSSION THREAD

**Discussion by doltae09** [bitcraze/discussions#599] [Q&A] [answered]
https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/parameters_tab/

I read the upside URL reference guidance. And then I design the simple parameter to change the fault injection data in "estimator_complementary.c".
I want to change the paramter in "Switch_fault" during real flight by joy stick mode.
Can you help me some tip about this problem?
![image](https://user-images.githubusercontent.com/79627957/222494077-ccc2afc9-a2d1-4de6-b0a1-6aeaf61cb2ed.png)

**Comment by knmcguire** [comment, marked as answer]
Hi!

So do you mean that the value in the table doesn't change or the default value? the Value in the left table should change since the default value is not the current value but the value that it initialized with. But it's strange that the default value is not showing anything.

Could you share the code of how you added this parameter to the firmware, with the value initialization?

**Reply by doltae09** [reply]
I have a miss take about using "param". Now It was solved by referencing the "rLimit" case.
![image](https://user-images.githubusercontent.com/79627957/222702167-9ae06543-bc56-4730-af52-4c40e63eb612.png)
![Uploading Screenshot from 2023-03-03 19-47-03.png…]()

**Reply by knmcguire** [reply]
I'm not sure what you mean with reference to the rlimit case, but at least form the screenshot I can see you using s_fault twice as the variable name, so you should remove one of those.

**Comment by redhammer1011** [comment]
I am having this issue as well while trying to change the motorPowerSet parameter

**Reply by doltae09** [reply]
In my case, I change the "PARAM_ADD" foam as follows:
![image](https://user-images.githubusercontent.com/79627957/222942498-ae6be5ac-b2e3-4f99-9ae2-f5051e8abec2.png)
I didn't follow the guideline so problem occur(cannot change the parameter's value in cfclient)
https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/userguides/logparam/

I don't know what kind of work in Crazyflie firmware with "motorPowerSet". If you want to change about "motorPowerSet" with your controller or estimator, try to make another value(i.e. motorPowerSet[0] = input[0]) and use the "PARAM_ADD" foam.
